nx.json

```diff
@@ -10,7 +10,7 @@
       "commitMessage": "chore(release): {projectName}-{version}"
     },
     "version": {
-      "preVersionCommand": "npx nx run-many -t build",
+      "preVersionCommand": "pnpm exec nx run-many -t build",
```

Comment:
It is not necessary to force `pnpm` here, since `npm` or `npx` will always be available and dies not rely on some lock files or something. When modules should not be installed, it will fetch nx, execute and fail.